### DISCUSSION TITLE
Failed to import SU2 in python

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [su2code/SU2#1910] [Q&A]
I am building `SU2` from source code, and the two steps are:

```
 python .\meson.py build --prefix=D:/SU2/bin -Denable-pywrapper=true
.\ninja.exe -C build install
```
The two steps work successfully. 

Then, I want to import `SU2` in python, but it reports:

```
>>> import SU2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'SU2'
```

How can I import `SU2` in python? Any suggestion is appreciated~~~

**Comment by pcarruscag** [comment]
Did you setup PATH and PYTHON_PATH according to what meson.py prints?

**Reply by zhang-qiang-github** [reply]
The `meson.py` printed information is:

![image](https://user-images.githubusercontent.com/29271051/216986023-2f677bd0-3189-48ab-957d-77d6702c5fd8.png)

And the PATH is modified as:

![image](https://user-images.githubusercontent.com/29271051/216987108-301082b8-d3c4-48a8-93a1-435154abf63a.png)

The PYTHON_PATH  is modified as:

![image](https://user-images.githubusercontent.com/29271051/216987395-b6959a3a-09f1-4b39-a4ab-3debdb2683e8.png)

When I import SU2, it report:

```
>>> import SU2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\SU2\bin\bin\SU2\__init__.py", line 14, in <module>
    from SU2 import run
  File "D:\SU2\bin\bin\SU2\run\__init__.py", line 3, in <module>
    from .interface import (
  File "D:\SU2\bin\bin\SU2\run\interface.py", line 41, in <module>
    SU2_RUN = os.environ['SU2_RUN']
  File "D:\Anaconda3\lib\os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'SU2_RUN'
```

**Reply by bigfooted** [reply]
You can check which environment variables are available in python, I guess this was not done correctly:

```
import os
for name, value in os.environ.items():
  print("{0}: {1}".format(name, value))
```

you can manually add the environment variable in python using
os.environ["SU2_RUN"] = "D:\SU2\bin\bin"

**Reply by zhang-qiang-github** [reply]
Thanks, it works after `os.environ["SU2_RUN"] = "D:\SU2\bin\bin"`
